server/api/documents.js

```diff
@@ -615,7 +625,9 @@ router.post('documents.update', auth(), async ctx => {
 
   // Update document
   if (title) document.title = title;
-  if (text) document.text = text;
+  //append to document
+  if (append) document.text += '\n' + text;
```

Comment:
Enforcing a newline here reduces the possibilities for this flag, It seems like it would be fine to give the option to the consumer of the API. Lets change this to use parenthesis to match the rest of the file/codebase.